src/components/Navbar/Navbar.jsx

```diff
@@ -1,14 +1,34 @@
 import React from 'react';
-import './Navbar.css';
+import './Navbar.scss';
+import { NavLink } from 'react-router-dom';
 import logo from '../../assets/ai.png';
 
+const links = [
+  { path: '/', text: 'Rockets' },
+  { path: 'missions', text: 'Missions' },
+  { path: 'profile', text: 'My Profile' },
+];
+
```

Comment:
👍 I like the idea to create an object of routes. 😄

Reply:
Thanks man 😄